### PR TITLE
Update PULL_REQUEST_TEMPLATE.md and CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by Akanksha1212**
### Description
I updated PR template and Contributing guidelines with the link to new testing guidelines so that new contributors can follow it every time they open a PR.

Fixes # [ISSUE]

### Type of Change:
**Delete irrelevant options.**
- Documentation

### How Has This Been Tested?
Describe the tests you ran to verify your changes. Provide instructions or GIFs so we can reproduce. List any relevant details for your test.

### Checklist:
**Delete irrelevant options.**

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [ ] I have commented on my code or provided relevant documentation, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [ ] Any dependent changes have been merged

**Code/Quality Assurance Only**
- [x] My changes generate no new warnings
- [ ] My PR currently breaks something (fix or feature that would cause existing functionality to not work as expected)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been published in downstream modules

### CODE REVIEW

**Review by nandini45 — APPROVED** [review]
i think those guidelines are definitely required. because these are some points which we repeat again and again.
if you want you can add mandatory too.😂

**Comment by nandini45** [comment]
@Akanksha1212  i think you should add one more guideline that if you working on a design when creating a PR do make sure to add a gif to show the responsiveness how your design is working

**Comment by Akanksha1212** [comment]
> @Akanksha1212 i think you should add one more guideline that if you working on a design when creating a PR do make sure to add a gif to show the responsiveness how your design is working
Sure. Will add that

File: docs/CONTRIBUTING.md [hunk 1]
```diff
@@ -7,6 +7,7 @@
 * When developing a new feature, include at least one test when applicable.
 * When submitting a PR, please follow [this template](.github/PULL_REQUEST_TEMPLATE/pull_request_template.md) (which will probably be already filled up once you create the PR).
 * When submitting a PR with changes to user interface (e.g.: new screen, ...), please add screenshots to the PR description.
+* When submitting a PR, please follow [testing guidelines](https://github.com/anitab-org/anitab-org.github.io/wiki/Prepare-testing-GitHub-Pages-for-your-PR), copy the link and place it in your PR.
```
Comment (isabelcosta) [inline docs/CONTRIBUTING.md:10]:
```suggestion
* When submitting a PR, please follow [testing guidelines](https://github.com/anitab-org/anitab-org.github.io/wiki/Prepare-testing-GitHub-Pages-for-your-PR), copy the link deployed site, and place it in your PR.
```

**Review by isabelcosta — APPROVED** [review]
I left a small comment, but I think this is fine :)
Great work @Akanksha1212

File: docs/CONTRIBUTING.md (hunk 1 above)
Comment (Akanksha1212) [reply docs/CONTRIBUTING.md:10]: Done

**Pull request merged**